src/core/public/chrome/ui/header/header_breadcrumbs.scss

```diff
@@ -13,43 +13,85 @@ $breadcrumbPolygon: polygon(
   0% 100%
 );
 
-.osdHeaderBreadcrumbs {
-  /*
-  filter defines a custom filter effect by grouping atomic filter primitives!
-  here we are using Gaussian filter with stdDeviation for applying
-  border-radius on clipped background.
-  */
-
-  filter: url("../../public/assets/round_filter.svg#round");
+/* remove background color on autofocus for euiBreadcrumbs in Popover */
+.euiBreadcrumbs__inPopover {
+  .euiLink.euiLink--text:focus {
+    background: none;
+  }
+}
 
-  .osdBreadcrumbs {
+.osdHeaderBreadcrumbs {
+  /* common for both light & dark theme */
+  .osdBreadcrumbs,
+  &--dark .osdBreadcrumbs {
     clip-path: $breadcrumbPolygon;
-    background-color: $osdHeaderBreadcrumbGrayBackground;
     padding: $euiSizeXS - 2.5 $euiSizeL - $euiSizeXS;
 
     &:first-child {
       clip-path: $firstBreadcrumbPolygon;
     }
+  }
+
+  .osdBreadcrumbs {
+    background-color: $osdHeaderBreadcrumbGrayBackground;
 
     &:last-child {
       background-color: $osdHeaderBreadcrumbBlueBackground;
     }
   }
 
-  .euiBreadcrumbSeparator {
-    display: none;
-  }
-
+  /* only light mode */
   .euiBreadcrumb__collapsedLink {
     color: $osdHeaderBreadcrumbCollapsedLink;
     background: $euiColorEmptyShade;
   }
 
-  .euiPopover__anchor {
-    padding: 0 $euiSizeS;
+  /* common for dark & light mode */
+  &,
+  &--dark {
+    /*
+  filter defines a custom filter effect by grouping atomic filter primitives!
+  here we are using Gaussian filter with stdDeviation for applying
+  border-radius on clipped background.
+  */
+    filter: url("../../public/assets/round_filter.svg#round");
+
+    button {
+      line-height: inherit;
+    }
+
+    .euiBreadcrumbSeparator {
+      display: none;
+    }
+
+    .euiPopover__anchor {
+      padding: 0 $euiSizeS;
+    }
+
+    .euiBreadcrumb:not(.euiBreadcrumb:last-child) {
+      margin-right: 0;
+    }
   }
 
-  .euiBreadcrumb:not(.euiBreadcrumb:last-child) {
-    margin-right: 0;
+  /* only dark mode */
+  &--dark {
+    .osdBreadcrumbs {
+      background-color: $osdHeaderMidnightSkyMediumBackground;
+      color: $osdHeaderMidnightSkyMediumLightColor;
+
+      &:hover {
+        color: $osdMidnightSkyMediumLightHoverColor;
+      }
+
+      &:last-child {
+        background-color: $osdHeaderPacificSkyDarkestBackground;
+        color: $euiColorFullShade;
+      }
+    }
+
+    .euiBreadcrumb__collapsedLink {
+      color: $euiColorGhost;
+      background: $euiColorEmptyShade;
+    }
```

Comment:
A screenshot of this in the description would be helpful.

Reply:
yes let me add